src/CognitiveServices/CognitiveServices.Test/ScenarioTests/CognitiveServicesAccountTests.cs

```diff
@@ -118,13 +125,7 @@ public void TestMinMaxAccountNames()
         {
             TestController.NewInstance.RunPsTest(traceInterceptor, "Test-MinMaxAccountName");
         }
-
-        [Fact]
-        [Trait(Category.AcceptanceType, Category.CheckIn)]
```

Comment:
Why are you removing this test?

Reply:
Rerunning this test is no longer possible as it goes over our max-limit of accounts per subscription. Also it doesn't add that much value as it's just creating 100 accounts and getting a count.

Reply:
Why did this work originally and not now?

Reply:
This was created when our test subscription didn't have that many accounts. To re-run this, we'd need to create a new subscription just for this specific test case. Also, this limit is placed by RP and is not handled specifically by the PS client so having this test case here in the first place feels weird. We have RP test cases elsewhere.